lib/src/schedules.dart

```diff
@@ -35,6 +35,55 @@ sealed class Schedule {
 
     return true;
   }
+
+  List<DateTime> findNextNOccurrences(Schedule schedule, int n,
+      {DateTime? fromDate, List<DateTime> excludeDates = const []}) {
+    final dates = <DateTime>[];
+    var currentDate = fromDate ?? DateTime.now();
+    excludeDates = excludeDates
+        .map((element) => DateTime(element.year, element.month, element.day))
```

Comment:
I have an extension to strip the time off of a `DateTime` object in the `extensions.dart` file. We could probably make it a public extension - what do you think?

Reply:
Or perhaps make it a separate utility function that is only available within the package?

Reply:
> I have an extension to strip the time off of a `DateTime` object in the `extensions.dart` file. We could probably make it a public extension - what do you think?

I saw it, but yes, it is private. Not sure about making it a public extension we should think a bit about that.

Reply:
I think we can make it a public extension. The extensions are not exported from the package, so I don't see any reason not to 👍🏻